src/components/RecipeDescription/RecipeDescription.test.jsx

```diff
@@ -0,0 +1,48 @@
+import { fireEvent, render } from '@testing-library/react';
+import RecipeDescription from './RecipeDescription';
+
+describe('<RecipeDescription/>', () => {
+  it('should render the recipe description', () => {
+    const recipeId = 1;
+    const description = 'some desc';
+    const editRecipeDescription = jest.fn();
+
+    const { getByText } = render(
+      <RecipeDescription
+        recipeId={recipeId}
+        description={description}
+        editRecipeDescription={editRecipeDescription}
+      />);
+
+    expect(getByText(description)).toBeVisible();
+  });
+
+  it('should edit the recipe description when modified with the EditDeleteBar', () => {
+    const recipeId = 1;
+    const description = 'some desc';
+    const editRecipeDescription = jest.fn();
+    const newDescription = 'some other description';
+
+    const { getByRole, getByText } = render(
+      <RecipeDescription
+        recipeId={recipeId}
+        description={description}
+        editRecipeDescription={editRecipeDescription}
+      />);
+
+    const descriptionComponent = getByText(description);
+    fireEvent.mouseOver(descriptionComponent);
+    const edit = getByRole('button', { name: /edit/i });
+    fireEvent.click(edit);
+    const editDescriptionTextbox = getByRole('textbox', { name: /edit/i }).querySelector('input');
```

Comment:
you could get the elements more easily if you asign a `data-testid` property to them
see https://testing-library.com/docs/queries/bytestid/

Reply:
+1

Reply:
About this topic, I came across some discussions on Stackoverflow regarding the use of `data-testid` vs `role` and `aria-label`. The points in favor of the later are:
- You are testing attributes that the final user will indeed see (that is, accessibility properties)
- The code is not adapting to the tests (as is the case with adding `data-testid` properties in jsx files)
On the other side, using `data-testid` selectors is more legible than using accessibility selectors. What do you think about this? What is the convention at TK?

Regarding the `querySelector('input')` at the end of this line, I think no option here would save us from that because of how Material UI is implemented. In few words, we can pass `data-testid` or `aria-label` to Material UI's `<Textbox>` but given it's a complex component wrapping an `<input>`, the properties won't be assigned to the `<input>` but to some parent element, hence the subsequent `querySelector('input')`. Do you by any chance know if there's a better way to solve this?

Reply:
The convention at TK is to use `data-testid` for components when possible.

Reply:
I see it can get a bit more complicated with a layer on top (like Material UI), but we use our own design system at TravelPerk (Suitcase), so I think that doesn't happen that often (?)

Reply:
Yes, like dani said we use `data-testid` more. The problem with material UI wouldn't happen on the suitcase components
Example:
https://github.com/travelperk/frontend/blob/84c7b93c4decf1ce48bacd2284c7a32e16c50fc4/apps/tk-web/src/screens/CompanySettings/screens/TravelPolicies/components/PoliciesFirstExperience/components/PoliciesNameStep.tsx#L49

and tests on:
https://github.com/travelperk/frontend/blob/84c7b93c4decf1ce48bacd2284c7a32e16c50fc4/apps/tk-web/src/screens/CompanySettings/screens/TravelPolicies/components/PoliciesFirstExperience/PoliciesFirstExperience.test.tsx#L185

Reply:
Great, will use `data-testid` in the future. Thanks guys!